File: packages/cli/tests/commands/template/fixtures/multi-stage/expected/python-async/build_dev.py
import asyncio
from e2b import AsyncTemplate, default_build_logger
from .template import template


async def main():
    await AsyncTemplate.build(
        template,
        alias="multi-stage-dev",
        on_build_logs=default_build_logger(),
    )


if __name__ == "__main__":
    asyncio.run(main())

File: packages/cli/tests/commands/template/fixtures/custom-commands/expected/typescript/build.dev.ts
import { Template, defaultBuildLogger } from 'e2b'
import { template } from './template'

async function main() {
  await Template.build(template, {
    alias: 'custom-app-dev',
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);